.bazelrc

```diff
@@ -13,6 +13,15 @@ build:linux --force_pic
 build:macos --force_pic
 build:clang-cl --compiler=clang-cl
 build:msvc --compiler=msvc-cl
+# `LC_ALL` and `LANG` is needed for cpp worker tests, because they will call "ray start".
+# If we don't hadd them, python's `click` library will raise an error.
+test --action_env=LC_ALL
+test --action_env=LANG
+# some envs needed by tests in pyenv
```

Comment:
maybe just put this in your own "~/.bazelrc" file, as other people might not use pyenv.